acceptance/analyzer_test.go

```diff
@@ -204,11 +204,12 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 		when("the provided layers directory isn't writeable", func() {
 			it("recursively chowns the directory", func() {
 				h.SkipIf(t, runtime.GOOS == "windows", "Not relevant on Windows")
-
 				output := h.DockerRun(t,
 					analyzeImage,
-					h.WithFlags("--env", "CNB_PLATFORM_API="+platformAPI),
-					h.WithBash(fmt.Sprintf("chown -R 9999:9999 /layers; chmod -R 775 /layers; %s some-image; ls -al /layers", analyzerPath)),
+					h.WithFlags("--env", "CNB_PLATFORM_API="+platformAPI,
+						"--network", registryNetwork),
+					h.WithBash(fmt.Sprintf("chown -R 9999:9999 /layers; chmod -R 775 /layers; %s %s; ls -al /layers", analyzerPath,
+						noAuthRegistry.RepoName("some-image"))),
```

Comment:
```suggestion
					h.WithFlags(
						"--env", "CNB_PLATFORM_API="+platformAPI,
						"--network", registryNetwork,
					),
					h.WithBash(
						fmt.Sprintf("chown -R 9999:9999 /layers; chmod -R 775 /layers; %s %s; ls -al /layers",
							analyzerPath,
							noAuthRegistry.RepoName("some-image")),
					),
```

Reply:
This was easier to parse in the IDE, but looking at it in GitHub I could not figure out what was going on!